packages/reakit/src/Step/__tests__/Step-test.tsx

```diff
@@ -1,14 +1,15 @@
-import React from "react";
+import * as React from "react";
 import { mount } from "enzyme";
-import Step from "../Step";
+import Step, { StepProps } from "../Step";
 import Hidden from "../../Hidden";
 
-const props = {
+const props: StepProps = {
   step: "foo",
   order: 0,
   register: jest.fn(),
   update: jest.fn(),
-  unregister: jest.fn()
+  unregister: jest.fn(),
+  isCurrent: jest.fn()
 };
 
 beforeEach(() => {
```

Comment:
Unfortunately i need that `[key: string]: any´ on the `StepProps` because of here, it starts to complain (i don't remember the error, i'm going to check later). I guess we can ignore this.